backfill/2021-09-07-error-geo/README.md

```diff
@@ -0,0 +1,325 @@
+    --experiments=shuffle_mode=service \
+    --region=us-central1 \
+    --usePublicIps=false \
+    --gcsUploadBufferSizeBytes=16777216 \
+"
+```
+
+```
+PROJECT="moz-fx-data-backfill-6"
+JOB_NAME="geo-backfill-stub-snapshot2"
+PIPELINE_FAMILY="stub_installer"
+
+## this script assumes it's being run from the ingestion-beam directory
+## of the gcp-ingestion repo.
+
+mvn clean compile exec:java -Dexec.mainClass=com.mozilla.telemetry.Decoder -Dexec.args="\
+    --runner=Dataflow \
+    --jobName=$JOB_NAME \
+    --project=$PROJECT \
+    --geoCityDatabase=gs://moz-fx-data-prod-geoip/GeoIP2-City/20210903/GeoIP2-City.mmdb \
+    --geoIspDatabase=gs://moz-fx-data-prod-geoip/GeoIP2-ISP/20210903/GeoIP2-ISP.mmdb \
+    --geoCityFilter=gs://moz-fx-data-prod-dataflow-templates/cities15000.txt \
+    --schemasLocation=gs://moz-fx-data-prod-dataflow/schemas/202109030341_302c86e7.tar.gz \
+    --inputType=bigquery_table \
+    --input=\"moz-fx-data-shared-prod:payload_bytes_raw.stub_installer_snapshot_bug1729069\" \
+    --bqReadMethod=storageapi \
+    --outputType=bigquery \
+    --bqRowRestriction=\"CAST(submission_timestamp AS DATE) BETWEEN '2021-08-10' AND '2021-08-19'\"
+    --bqWriteMethod=file_loads \
+    --bqClusteringFields=submission_timestamp \
+    --output=${PROJECT}:\${document_namespace}_live.\${document_type}_v\${document_version}_from_snapshot2 \
+    --errorOutputType=bigquery \
+    --errorOutput=${PROJECT}:payload_bytes_error.${PIPELINE_FAMILY} \
+    --experiments=shuffle_mode=service \
+    --region=us-central1 \
+    --usePublicIps=false \
+    --gcsUploadBufferSizeBytes=16777216 \
+"
+```
+
+## Telemetry and structured
+
+See the various scripts in this directory.
+
+## Cleanup
+
+```
+# BE CAREFUL! This removes all BQ datasets and GCS buckets from the target project,
+# including their contents; it cannot be undone.
+bq ls --project_id=moz-fx-data-backfill-6 | tail -n+3 | awk '{print $1}' | xargs -I{} -n1 echo bq rm -r -f "moz-fx-data-backfill-6:{}"
+gsutil ls -p moz-fx-data-backfill-6 | xargs echo gsutil -m rm -r
+```
+
+We also did some additional cleanup for other backfill projects:
+```
+# BE CAREFUL! This removes all BQ datasets and GCS buckets from the target project,
+# including their contents; it cannot be undone.
+bq ls --project_id=moz-fx-data-backfill-7 | tail -n+3 | awk '{print $1}' | xargs -I{} -n1 echo bq rm -r -f "moz-fx-data-backfill-7:{}"
+gsutil ls -p moz-fx-data-backfill-7 | xargs echo gsutil -m rm -r
+bq ls --project_id=moz-fx-data-backfill-8 | tail -n+3 | awk '{print $1}' | xargs -I{} -n1 echo bq rm -r -f "moz-fx-data-backfill-8:{}"
+gsutil ls -p moz-fx-data-backfill-8 | xargs echo gsutil -m rm -r
+bq ls --project_id=moz-fx-data-backfill-17 | tail -n+3 | awk '{print $1}' | xargs -I{} -n1 echo bq rm -r -f "moz-fx-data-backfill-17:{}"
+```
+
+An admin needs to run:
+```
+bq rm moz-fx-data-shared-prod:payload_bytes_raw.stub_installer_snapshot_bug1729069
```

Comment:
I'm ready for @jasonthomas to run this command when convenient. I've cleaned up all the tables in `moz-fx-data-backfill-6` that I have permissions to delete.

Reply:
Done.